theme_buzzy/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" inherit_id="website.s_numbers_showcase">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc2" separator=" "/>
+    </xpath>
+    <!-- Grid items -->
+    <xpath expr="//div[hasclass('o_grid_item')]" position="attributes">
+        <attribute name="class" add="justify-content-center" remove="justify-content-start" separator=" "/>
+    </xpath>
```

Comment:
If you remove the option in the community PR, don't forget to remove this customization :wink: